python/tk_multi_about/dialog.py

```diff
@@ -106,7 +106,16 @@ def show_in_fs(self):
             if not QtGui.QDesktopServices.openUrl(
                 "file://{0}".format(disk_location).replace("\\", "/")
             ):
-                self._app.log_error("Failed to open '%s'!" % disk_location)
+                # You will have a False return here on Windows when using
+                # drive letters and not UNC paths - add third slash
+                if not QtGui.QDesktopServices.openUrl(
+                    "file:///{0}".format(disk_location).replace("\\", "/")
+                ):
+                    self._app.log_error(
+                        "Failed to open system file browser for '%s'!" % disk_location
+                    )
+                    break
```

Comment:
Adding a break in here changes the behaviour I think. The comment on line 103 says we are launching a window for each location, but adding a break would mean we would stop trying to open locations if we found one locations that  didn't open.
I think it seems reasonable to try and open other locations even if one of them failed, so maybe we shouldn't have a break here, what do you think?

Reply:
I totally agree @pscadding and will make that change. What do you think of the added log_info line right below it? Do you think we should keep it as info or set it up as a debug log only? Thanks!

Reply:
Yeah I considered that line as well, but tbh I'm indifferent, its a low volume operation so its not likely to clog up logs and isn't going to impact on performance. 
If you think there is value in being always logged then I'm fine with it being info.